### ISSUE TITLE
[BUG] You can win the game by flagging all spaces

### ISSUE THREAD

**Issue by anuvogelat** [LyrasaurusRose/SweepMine#2]
**Describe the bug**
If you right click on all the spaces the game will end with a message that you won.

**Expected behavior**
I expected it to do nothing: since I didn't flag all the mines correctly (I overflagged) it should just keep running.

**What version of the addon are you using?**
 - Version: 1.0.1

**Please type `/dump SweepBoard` in the chat and attach the results here**
SweepBoard Dump: 
```
SweepBoard={
  [1]={
    [1]=2,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=2,
    [6]=2,
    [7]=2,
    [8]=0
  },
  [2]={
    [1]=2,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=2,
    [6]=3,
    [7]=2,
    [8]=0
  },
  [3]={
    [1]=2,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=2,
    [6]=2,
    [7]=2,
    [8]=0
  },
  [4]={
    [1]=2,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=2,
    [6]=2,
    [7]=3,
    [8]=3
  },
  [5]={
    [1]=3,
    [2]=3,
    [3]=3,
    [4]=2,
    [5]=2,
    [6]=2,
    [7]=3,
    [8]=2
  },
  [6]={
    [1]=2,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=2,
    [6]=2,
    [7]=2,
    [8]=2
  },
  [7]={
    [1]=2,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=3,
    [6]=2,
    [7]=3,
    [8]=2
  },
  [8]={
    [1]=3,
    [2]=2,
    [3]=2,
    [4]=2,
    [5]=2,
    [6]=3,
    [7]=2,
    [8]=2
  },
}
```

**If your issue is with bombs, please also include `/dump SweepBombs`**
SweepBombs Dump:
```code```

**Additional context**
Originally discovered when I was porting the code and testing whether the port worked.

**Comment by buzzyjessibee** [comment]
This is due to me trying to take the easy way out - oops! (never works in coding)
Working on it for v1.1.0 as well as other QoL features! Thank you for reporting!

**Comment by buzzyjessibee** [comment]
Fixed in release v1.1.0
